lua/crates/command.lua

```diff
@@ -1,41 +1,37 @@
-local actions = require("crates.actions")
```

Comment:
Inline these wrapped around a function to lazily require the modules😅 This seems to be improve the setup time from around 3ms to 0.5 ish for me